docs/submission/validating-molecular-data.md

```diff
@@ -251,12 +249,15 @@ Depending on the type of sequencing that was done, the `read_groups` section of
 Sequencing data of both `BAM` and `FASTQ` type files are accepted, with the exception of interleaved FASTQ files.  Metadata payloads like the example above must follow these data validations:
 
 1. Compression of FASTQ files is **required**; both _gzip_ (suffix .fq.gz or .fastq.gz) or _bz2_ (suffix .fq.bz2 or .fastq.bz2) are supported.
-1. For both FASTQ and BAM submission, all files in the `files` section must be unique in the payload.
-1. All `read_groups` in the payload and BAM header must belong to a single sample.
-1. `platform_units` must be unique with a one-to-one relationship with `submitter_read_group_id`.
+1. For both FASTQ and BAM submissions, all files in the `files` section must be unique in the payload.
+1. All `read_groups` in the payload and BAM header must belong to a single sample. 
+1. All `submitter_read_group_id` must be unique across the payload. 
+1. `read_group_id_in_bam` must be unique within a single BAM. 
+1. `platform_units` must be unique across the payload with a one-to-one relationship with `submitter_read_group_id`.
 1. The total number of `read_group` objects must match the number specified in `read_group_count`.
-1. For paired-end sequencing, both `file_r1` and `file_r2` are required.  For single-end sequencing, only `file_r1` is required (`file_r2` must be populated with `null`).
-1. For paired-end sequencing, `file_r1` and `file_r2` must not be the same file. 
+1. For FASTQ paired-end sequencing, both `file_r1` and `file_r2` are required. `file_r1` and `file_r2` must not be the same file. 
+1. For BAM paired-end sequencing, both `file_r1` and `file_r2` are required.  These must be the same file. 
```

Comment:
You have to submit the same file twice?

Reply:
yes its a paired-end thing.  the same file isnt submitted twice, but it is in teh fields r1/r1 as the data for 2 ends is in one file

Reply:
Must be the same file `reference`?

Reply:
hmm no i think thats confusing.  they must be the same file.  this is about read-groups and the metadata that goes in there, so i think its best to be explicit about it being the same file.